src/archivematicaCommon/lib/env_configparser.py

```diff
@@ -0,0 +1,73 @@
+import os
+import ConfigParser
+import functools
+
+
+def fallback_option(fn):
```

Comment:
I like this! Maybe this is too forward thinking, but how does it interact with the Python3 configparser's fallback?